### ISSUE TITLE
add on do not start , after  update .

### ISSUE THREAD

**Issue by gnkarn** [hassio-addons/addon-tasmoadmin#9]
@frenck  there is a problem on the latest version 

```
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to INFO
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] 02-updates.sh: executing... 
INFO: You are running the latest version of this add-on
[cont-init.d] 02-updates.sh: exited 0.
[cont-init.d] 10-requirements.sh: executing... 
[cont-init.d] 10-requirements.sh: exited 0.
[cont-init.d] 11-nginx.sh: executing... 
[cont-init.d] 11-nginx.sh: exited 0.
[cont-init.d] 12-migrate.sh: executing... 
/var/run/s6/etc/cont-init.d/12-migrate.sh: line 10: hass.log.inf: command not found
[cont-init.d] 12-migrate.sh: exited 127.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.
 
 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
```

**Comment by ascillato** [comment]
Hi,

I have just update and works for me.

Try uninstall the add on and install again. Please be aware of installing TasmoAdmin not SonWeb.

**Comment by frenck** [comment]
This is a bug... I’m so sorry for this, will fix this first thing in the morning!

**Comment by frenck** [comment]
This has been fixed in `v0.2.1`.
